addons/mail/static/src/core/common/composer.xml

```diff
@@ -35,8 +35,8 @@
                 <div class="d-flex o-mail-Composer-bg flex-grow-1 border"
                     t-att-class="{
                         'o-mail-Composer-compactContainer m-1 shadow-sm border-secondary': compact and !props.composer.message,
-                        'o-mobile rounded-3': isMobileOS,
-                        'rounded-3' : normal,
+                        'o-ios-mobile': isIosApp,
+                        'rounded-3' : normal or isMobileOS,
                         'rounded-3 align-self-stretch' : extended,
```

Comment:
Nitpick: can remove the `rounded-3` in the extended condition, and put `or extended` in the `rounded-3` self conditional part.